packages/components/src/components/hds/dropdown/list-item/checkbox.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { getElementId } from '../../../../utils/hds-get-element-id.ts';
+import type { FlightIconSignature } from '@hashicorp/ember-flight-icons/components/flight-icon';
+import type { HdsFormCheckboxBaseSignature } from '../../form/checkbox/base.ts';
+
+export interface HdsDropdownListItemCheckboxSignature {
+  Args: HdsFormCheckboxBaseSignature['Args'] & {
+    count?: string | number;
```

Comment:
notice: in the documentation we say it's a `string` only. One of the two needs to be updated: https://helios.hashicorp.design/components/dropdown?tab=code#dcheckbox

Reply:
yes, I will update the docs. learning from the recent feedback it made sense for us to allow numbers too